tests/commands/record/test_session.py

```diff
@@ -6,9 +7,17 @@
 
 
 class SessionTest(CliTestCase):
+    """
+    This test needs to specify `clear=True` in mocking because the test is run on GithubActions. 
+    Otherwise GithubActions will export $GITHUB_* variables at runs.
+    """
 
     @responses.activate
-    @mock.patch.dict(os.environ, {"LAUNCHABLE_TOKEN": CliTestCase.launchable_token})
+    @mock.patch.dict(os.environ, {
+        "LAUNCHABLE_TOKEN": CliTestCase.launchable_token,
+        # it's needed to run the CLI because the line cleared all exsting environment
+        'LANG': 'C.UTF-8',
```

Comment:
Q: Why do we need LANG env?

Reply:
Because `CliRunner().invoke(command)` needs the flag. In other cases, the flag is automatically added by the shell. But in this case, `clear=True` removes it. See the run result that is without `LANG=C.UTF-8` https://github.com/launchableinc/cli/runs/7442868706?check_suite_focus=true#step:10:102 .